keyboards/mechwild/sugarglider/keymaps/default/rules.mk

```
@@ -0,0 +1 @@
+ENCODER_MAP_ENABLE = yes
```

The new [tri layer feature](https://docs.qmk.fm/#/feature_tri_layer) can be used here.
```suggestion
ENCODER_MAP_ENABLE = yes
TRI_LAYER_ENABLE = yes
```